php/WP_CLI/Runner.php

```diff
@@ -258,16 +259,20 @@ public function find_command_to_run( $args ) {
 				if ( count( $cmd_path ) > 1 ) {
 					$child = array_pop( $cmd_path );
 					$parent_name = implode( ' ', $cmd_path );
+					$suggestion = $this->get_subcommand_suggestion( $child, $command );
 					return sprintf(
-						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.",
+						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.%s",
 						$child,
 						$parent_name,
-						$parent_name
+						$parent_name,
+						! empty( $suggestion ) ? "\n\nDid you mean '{$suggestion}'?" : ''
```

Comment:
Your second variant probably reads better for humans, but I think it might make parsing of the result unnecessarily complicated.